test/house_test.rb

```diff
@@ -0,0 +1,110 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/room'
+require './lib/house'
+
+class RoomTest < Minitest::Test
+  def test_it_exists
+    house = House.new("$400000", "123 sugar lane")
+
+    assert_instance_of House, house
+  end
+
+  def test_it_has_a_price_and_address
+    house = House.new("$400000", "123 sugar lane")
+
+    assert_equal 400000, house.price
+    assert_equal "123 sugar lane",house.address
+  end
+
+  def test_rooms_can_be_added
+    house = House.new("$400000", "123 sugar lane")
+    room_1 = Room.new(:bedroom, 10, '13')
+    room_2 = Room.new(:bedroom, 11, '15')
+
+    assert_equal [], house.rooms
+
+    house.add_room(room_1)
+    house.add_room(room_2)
+    assert_equal [room_1, room_2], house.rooms
+  end
+
+  def test_it_can_be_above_or_below_market_average
+    house = House.new("$400000", "123 sugar lane")
+
+    assert_equal false, house.above_market_average?
+  end
+
+  def test_it_can_return_rooms_by_category
+    house = House.new("$400000", "123 sugar lane")
+    room_1 = Room.new(:bedroom, 10, '13')
+    room_2 = Room.new(:bedroom, 11, '15')
+    room_3 = Room.new(:living_room, 25, '15')
+    room_4 = Room.new(:basement, 30, '41')
+    house.add_room(room_1)
+    house.add_room(room_2)
+    house.add_room(room_3)
+    house.add_room(room_4)
+
+    assert_equal [room_1, room_2],house.rooms_from_category(:bedroom)
+    assert_equal [room_4], house.rooms_from_category(:basement)
+  end
+
+  def test_it_can_return_total_area_of_house
+    house = House.new("$400000", "123 sugar lane")
+    room_1 = Room.new(:bedroom, 10, '13')
+    room_2 = Room.new(:bedroom, 11, '15')
+    room_3 = Room.new(:living_room, 25, '15')
+    room_4 = Room.new(:basement, 30, '41')
+    house.add_room(room_1)
+    house.add_room(room_2)
+    house.add_room(room_3)
+    house.add_room(room_4)
+
+    assert_equal 1900, house.area
+  end
+
+  def test_show_details
+    house = House.new("$400000", "123 sugar lane")
+    house.details
```

Comment:
this doesn't need to be here-- you're calling `house.details` in your assertion on line 73, so this is just calling it an extra time for no particular reason.  Although it's unnecessary here, it doesn't hurt you, but in some cases, it can really mess up your results, so best to just avoid it altogether.

Reply:
I was PRYing at that point and had it there to hit a pry.  Forgot to take it out.  I'm sure one of these days I'm going to destroy my code doing that and not know why.  I'll remember your warning on that day!